Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM golang:1.21-bookworm
 
 ARG TAMAGO_VERSION
-ARG FT_LOG_ORIGIN
```

Comment:
What's the motivation behind renaming this arg but not the URL and bin?

Reply:
Other repos call this `LOG_ORIGIN` so I wanted to try to have common vars be named the same across all repos. Can remove the `FT_` prefix on the others if you want in a follow up?